modules/nw-enabling-a-provisioning-network-after-installation.adoc

```diff
@@ -15,22 +15,22 @@ In {product-title} 4.8 and later, you can enable a `provisioning` network after
 * A dedicated physical network must exist, connected to all worker and control plane nodes.
 * You must isolate the native, untagged physical network.
 * The network cannot have a DHCP server when the `provisioningNetwork` configuration setting is set to `Managed`.
-* You must connect the control plane nodes to the network with the same network interface, such as `eth0` or `eno1`.
+* You may omit the `provisioningInterface` setting in {product-title} 4.9 to use the `bootMACAddress` configuration setting.
 
 .Procedure
 
-. Identify the provisioning interface name for the cluster nodes. For example, `eth0` or `eno1`.
+. When setting the `provisioningInterface` setting, first identify the provisioning interface name for the cluster nodes. For example, `eth0` or `eno1`.
 
 . Enable the Preboot eXecution Environment (PXE) on the `provisioning` network interface of the cluster nodes.
 
-. Retrieve the current state of the `provisioning` network and save it to a provisioning configuration resource file:
+. Retrieve the current state of the `provisioning` network and save it to a provisioning custom resource file:
```

Comment:
We often spell this as "custom resource (CR)" on first use and then "CR" in subsequent uses in a module, but your call.